examples/hotsos-example-openstack.summary.yaml

```diff
@@ -109,8 +109,7 @@ openstack:
     - qemu-kvm 1:4.2-3ubuntu6.19
     - radvd 1:2.17-2
   neutron-l3ha:
-    backup:
-      - 984c22fd-64b3-4fa1-8ddd-87090f401ce5
+    backup: 1
```

Comment:
I have no context here but shouldn't it be better if `backup` also contain a count/uuids list like `running` for the sake of uniformity?

Reply:
i guess we could argue that having a long list of uuids for vms is also superfluous but it does generally tend to be more manageable